content/docs/app-developer-guide/build-an-arm-app.md

```diff
@@ -0,0 +1,92 @@
++++
+title="Build an ARM app"
+weight=2
+summary="The basics of taking your app from source code to runnable ARM image."
++++
+
+As of today, there are no CNB platforms (e.g. [pack][pack]) that support building ARM application images. In the following tutorial, we will be performing a build "manually", in that we will be performing a build by invoking the lifecycle directly.
+
+### 1. Prepare your working directory
+
+On your Linux ARM machine with a [docker][docker] daemon installed, prepare the following directory tree structure.
+
+```bash
+tree ~/workspace/
+~/workspace/
+├── buildpacks
+│   └── hello-world
+└── platform
+```
+
+In addition, clone the [samples][samples] repository which will contain the application source code.
+
+```bash
+# clone the repo
+git clone https://github.com/buildpacks/samples ~/workspace/samples
+```
+
+### 2. Prepare the assets
+
+Now we need to prepare assets that will be used during the build process.
+
+First we download and extract the [lifecycle][lifecycle] release, compiled for ARM. Make sure to replace `<RELEASE-VERSION>` with a valid release version.
+
+```bash
+# change to destination directory
+cd ~/workspace
+
+# download and extract lifecycle
+curl -L https://github.com/buildpacks/lifecycle/releases/download/v<RELEASE-VERSION>/lifecycle-v<RELEASE-VERSION>+linux.arm64.tgz | tar xf -
+```
+
+Next we make sure that our buildpack directory is structured in a way that the lifecycle will expect.
+
+```bash
+# copy hello-world buildpack
+cp -R ~/workspace/samples/buildpacks/hello-world ~/workspace/buildpacks/hello-world/0.0.1
+```
+
+And finally we write the `order.toml` file that references the hello-world buildpack.
+
+```bash
+cat > ~/workspace/order.toml <EOF
+[[order]]
+[[order.group]]
+id = "samples/hello-world"
+version = "0.0.1"
+optional = false
+EOF
+```
+
+### 3. Build your app
+
+Now we can build our app. For this example we'll be using the docker CLI to invoke the lifecycle directly.
```

Comment:
```suggestion
Now we can build our app. For this example we will be using the docker CLI to invoke the lifecycle directly.
```

I'm not sure but I think that in most of the docs we aren't using the long form.

Reply:
_Fixed in latest commit_.